klipper-helm.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: klipper-helm
+  version: 0.8.0-build20230510
```

Comment:
cc @rawlingsj if you know any wizardry for getting updates working with this version schema...

Reply:
fwiw the lint that's failing can be recreated locally with
```
wolfictl lint klipper-helm.yaml
```